libs/clj-icat-direct/src/clj_icat_direct/queries.clj

```diff
@@ -517,21 +517,19 @@
 
 (def queries
   {:count-all-items-under-folder
-   "WITH user_groups AS ( SELECT g.*
+   "WITH user_groups AS ( SELECT g.group_user_id
                             FROM r_user_main u
                             JOIN r_user_group g ON g.user_id = u.user_id
                            WHERE u.user_name = ?
                              AND u.zone_name = ? ),
 
-         parent      AS ( SELECT * from r_coll_main
+         parent      AS ( SELECT DISTINCT coll_id, coll_name from r_coll_main
```

Comment:
`coll_id` is unique in `r_coll_main`, so you don't need to use `DISTINCT`. It will only slow the query down as duplicates that don't exist will be searched for.